tdmpc2/common/buffer.py

```diff
@@ -41,8 +41,8 @@ def _reserve_buffer(self, storage):
 		return ReplayBuffer(
 			storage=storage,
 			sampler=self._sampler,
-			pin_memory=True,
-			prefetch=1,
```

Comment:
disabling prefetch bc I haven't tested how multithreaded cuda ops work while executing a cudagraph.
Also pin_memory incur some overhead, best to avoid it
https://docs-preview.pytorch.org/pytorch/tutorials/2983/intermediate/pinmem_nonblock.html